pkg/backend/openshiftcluster.go

```diff
@@ -215,10 +235,14 @@ func (ocb *openShiftClusterBackend) updateAsyncOperation(ctx context.Context, lo
 				// asyncOperations errors. Otherwise - return generic error
 				err, ok := backendErr.(*api.CloudError)
 				if ok {
-					log.Print(backendErr)
+					if err.CloudErrorBody.Category != "" && err.CloudErrorBody.Category == api.AROUserError {
+						resultType = 1
```

Comment:
Could you please add the result type as a constant? This would make the code more easily understandable as we have:

```
0 == sucess
1 == user error
2 == internal server error
```

Thank you.

Reply:
Created a resultType constant as mentioned, and changed to be a string instead of int.